plugins/node/opentelemetry-instrumentation-redis/package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "@opentelemetry/instrumentation-redis",
   "version": "0.49.0",
-  "description": "OpenTelemetry instrumentation for `redis` v2 and v3 database client for Redis",
+  "description": "OpenTelemetry instrumentation for `redis` database client for Redis",
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js-contrib",
   "scripts": {
-    "test": "nyc mocha 'test/**/*.test.ts'",
+    "test": "npm run test-v2-v3 && npm run test-v4",
+    "test-v2-v3": "tav redis 3.1.2 npm run test-v2-v3-run",
+    "test-v4": "tav redis 4.7.1 npm run test-v4-run",
+    "test-v2-v3-run": "nyc mocha --no-clean --require '@opentelemetry/contrib-test-utils' 'test/v2-v3/*.test.ts'",
+    "test-v4-run": "nyc mocha --no-clean --require '@opentelemetry/contrib-test-utils' 'test/v4/*.test.ts'",
```

Comment:
reviewer note: 

this is aligned with [`@opentelemetry/instrumentation-mongodb`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/11a94b09111787489643365690591d1e607dd119/plugins/node/opentelemetry-instrumentation-mongodb/package.json#L10)

Reply:
IMO we should be leaving testing of multiple versions of the target library to `test-all-versions` tests -- that's what they are for. I think the mongodb testing change was misguided.

I have an alternative PR that builds on yours that shows what I mean: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2915
That PR also has a few other changes. PTAL.